### ISSUE TITLE
Train dataset for LLMxCPG-D model

### ISSUE THREAD

**Issue by TheLiao233** [qcri/llmxcpg#2]
Hello, I only saw `llmxcpg_query_train.json` in the `data` folder. It seems that it is only used to train the `LLMxCPG-Q` model. Could you please provide a dataset that can be directly used to train the `LLMxCPG-D` model?

**Comment by TheLiao233** [comment]
Hello, I used the `llmxcpg_detect_train.json` dataset you uploaded to train the `Qwen2.5-Coder-14B-Instruct` model, and the training loss has been reduced to a very low level.  However, when I used it to test the public dataset in the `data` folder, I found that it had no effect at all.  May I ask why this is the case?

Could it be that the training dataset is incomplete or there is a problem with the dataset used for testing?

<img width="3033" height="648" alt="Image" src="https://github.com/user-attachments/assets/e5a281e4-1ef8-44f0-96c5-2c609643ea75" />

<img width="3038" height="789" alt="Image" src="https://github.com/user-attachments/assets/af1967ff-6723-40f9-8062-cdce4efdb6bc" />
